docs/script-api.md

```diff
@@ -374,9 +374,11 @@ Get the [ParticleDB](#particledb) details of the specified ID
 ### [`set_drop_chance`](https://github.com/spelunky-fyi/overlunky/search?l=Lua&q=set_drop_chance)
 #### Params: `int dropchance_id, int new_drop_chance`
 Alters the drop chance for the provided monster-item combination (use e.g. set_drop_chance(DROPCHANCE.MOLE_MATTOCK, 10) for a 1 in 10 chance)
+void set_drop_chance(uint16_t dropchance_id, uint32_t new_drop_chance)
```

Comment:
So this was me being stupid apparently. I don't know exactly why I thought what I said in the last comment about this, but the auto-docs were actually doing their job well and I just asked you to introduce a bug the in the docs  🤦 
So you can take the comment in script.cpp away again. Sorry about that